corebehrt/data/prepare_data.py

```diff
@@ -188,6 +188,7 @@ def _prepare_mlm_features(self, predefined_splits, val_ratio=0.2) -> Data:
         data_cfg = self.cfg.data
         model_cfg = self.cfg.model
         paths_cfg = self.cfg.paths
+        paths_cfg = self.cfg.paths
```

Comment:
double?

Reply:
adressed in 81aba9ebe7b955d6f86e36d2783a7c2dae12cb9a